### PR TITLE
[Lands] Create homepage nav icons

### PR DESCRIPTION
**Pull request by KhadijaMahanga**
## Description

Create homepage nav icons

Fixes # (issue)

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Screenshots
screenshot from 2018-11-01 16-10-46

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: hurumap_land/templates/homepage.html [hunk 1]
```diff
@@ -23,35 +23,78 @@ <h4>{{ HURUMAP.name }} provides useful facts and data about land ownership in So
...
-  <div class="column-third">
-    <p><span class="img-bg rounded bg-color-primary"><img src="{% static 'img/icons/blurbs/analysis.svg' %}"/></span></p>
-    <h3>Visualize</h3>
-    <p>Our library of charts gives you insight into land data from provinces you research. Look for them on profile pages. You can embed the charts on your own site. </p>
+  <div class="home-nav text-center section-container bg-color-white">
+    <ul>
+      <li><a>
+          <span class="img-bg rounded text-white" style="padding: 20px 27px;">
+          <img src="{% static 'img/icons/blurbs/papyrus.svg'%}"/>
+          <br/>ABOUT</span></a></li>
```
Comment (DavidLemayian) [inline hurumap_land/templates/homepage.html]: This is missing a link to about page or how it works?

**Review by DavidLemayian — CHANGES_REQUESTED** [review]
https://github.com/CodeForAfrica/HURUmap-apps/pull/97/files#r230137168

File: hurumap_land/templates/homepage.html (hunk 1 above)
Comment (KhadijaMahanga) [reply hurumap_land/templates/homepage.html]: @DavidLemayian I have directed the link to about page

**Review by DavidLemayian — DISMISSED** [review]
Thanks @KhadijaMahanga. Please introduce the `rgb(172,206,217)` border that comes between map and first section.

**Comment by KhadijaMahanga** [comment]
@DavidLemayian there's a border with color `#a5d0dc` which is similar to `rgb(172,206,217)` ; but the ocean in the map has similar color too. Thats why it's not visible

**Pull request merged**